packages/framework/composer.json

```diff
@@ -68,12 +68,12 @@
         "psr/log": "^1.0",
         "sensio/distribution-bundle": "^5.0.21",
         "sensio/framework-extra-bundle": "^3.0.29",
-        "shopsys/doctrine-orm": "2.6.2",
+        "shopsys/doctrine-orm": "^2.6.3",
         "shopsys/jparser": "0.1",
         "shopsys/postgres-search-bundle": "0.1",
-        "shopsys/migrations": "dev-master",
-        "shopsys/form-types-bundle": "dev-master",
-        "shopsys/plugin-interface": "dev-master",
+        "shopsys/migrations": "7.0.0-beta3",
+        "shopsys/form-types-bundle": "7.0.0-beta3",
+        "shopsys/plugin-interface": "7.0.0-beta3",
         "snc/redis-bundle": "2.1.4",
```

Comment:
very strange, not event human factor but also computer factor forgets to fix this package with `^` notation https://github.com/shopsys/shopsys/issues/549